### PR TITLE
chore: remove feature flags

### PR DESCRIPTION
**Pull request by berzerkeer**
### Description
 - Removes feature flags for usage & billing.
 - moves pricing url to constant.

### CODE REVIEW

File: app/client/src/utils/hooks/useOnUpgrade.ts [hunk 1]
```diff
@@ -1,45 +1,29 @@
...
   intercomMessage?: string;
   logEventName?: EventName;
   logEventData?: any;
 };
 
-const useOnUpgrade = (props: Props) => {
-  const { intercomMessage, logEventData, logEventName } = props;
-  const features = useSelector(selectFeatureFlags);
-  const instanceId = useSelector(getInstanceId);
+const PRICING_PAGE_URL =
```
Comment (sbalaji1192) [inline app/client/src/utils/hooks/useOnUpgrade.ts:11]: move this to the file where we have ASSET_CDN_URL

File: app/client/src/utils/hooks/useOnUpgrade.ts [hunk 2]
```diff
@@ -1,45 +1,29 @@
...
-    if (features.USAGE_AND_BILLING) {
-      window.open(
-        `https://www.appsmith.com/api/preview?secret=8JPsJRnSkt6Va8FzxUPFhZezxZuHRnSU&slug=pricing-preview?source=CE&instance=${instanceId}`,
-        "_blank",
-      );
-    } else {
-      triggerIntercom(intercomMessage || createMessage(UPGRADE_TO_EE_GENERIC));
-    }
+    window.open(
+      `${PRICING_PAGE_URL}?source=CE&instance=${tenantConfig?.instanceId}`,
```
Comment (sbalaji1192) [inline app/client/src/utils/hooks/useOnUpgrade.ts:24]: move the entire thing to the constant

Comment (berzerkeer) [reply app/client/src/utils/hooks/useOnUpgrade.ts:24]: The params too ? The params take value from the hook, only the base URL is going to be the same. So shouldn't we just move the base the URL to constant and append the params in the file from where we are actually redirecting to the URL.

Comment (sbalaji1192) [reply app/client/src/utils/hooks/useOnUpgrade.ts:24]: create a function that accepts this value and returns the constant file.

Comment (berzerkeer) [reply app/client/src/utils/hooks/useOnUpgrade.ts:24]: Yeah thats fine, but do we know whether this pricing URL will always take `source` and `instance` as search params?
Even though we can conditionally append or not, it just adds more checks . Thats why I was wondering.

Comment (sbalaji1192) [reply app/client/src/utils/hooks/useOnUpgrade.ts:24]: Let add those two keys always for now. Will think about later how to do that conditionally

Comment (berzerkeer) [reply app/client/src/utils/hooks/useOnUpgrade.ts:24]: Sure 👍

**Review by sbalaji1192 — APPROVED** [review]

**Pull request merged**